### PR TITLE
further enhancements for download page to enable more visibility betw…

### PR DESCRIPTION
**Pull request by mswiderski**
…een community and product

screen shot 2018-07-12 at 13 03 48
screen shot 2018-07-12 at 13 03 57

### CODE REVIEW

File: download/download.html.haml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+      * WildFly server
+      * jBPM console
+      * KIE Server
+      * jBPM Case management showcase app
+      * jBPM Service repository (with community work items)
+
+      all configured to work smoothly together.
+
+      Once download is compete have a look at link:../learn/gettingStarted.html[*Get started guide*]
```
Comment (krisv) [inline download/download.html.haml]: comp*l*ete

File: download/download.html.haml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+      |*Help* | |
+      |Roadmap impact |image:checkYes.png[Yes] Use link:../community/getHelp.html[our forum], #{site.issueTracker}[issue tracker] or send a link:../code/sourceCode.html[Pull Request] |image:checkYes.png[Yes] Prioritized above community requests
+      |Public questions |image:checkYes.png[Yes] Use link:../community/getHelp.html[StackOverflow] |image:checkYes.png[Yes] Or call 24/7 support instead
+      |Private 24/7 support |image:checkNo.png[No] |image:checkYes.png[Yes] https://access.redhat.com/support/contact/[By web or phone], guaranteed response time (SLA)
+      |Red Hat consultancy, training and POC assistance |image:checkNo.png[No] |image:checkYes.png[Yes] Talk to your account manager
+
+      |*Release schedule* | |
+      |Beta/CR releases |image:checkYes.png[Yes] Usually every few weeks |image:checkYes.png[Yes] After Final community release
+      |Stable releases |image:checkYes.png[Yes] Final usually every 3 months |image:checkYes.png[Yes] GA usually 2 months after community Final release (if it is productized)
```
Comment (krisv) [inline download/download.html.haml]: Could you maybe update this to the new reality? remove Beta/CR for community, update releases to every 3 weeks, for product releases every 3 months?

File: _config/site.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 encoding: UTF-8
 title: jBPM - Open Source Business Automation Toolkit
-description: jBPM is a toolkit for building business applications backed by solid business automation capabilities.
-keywords: business automation, business process management, case management, business rules, decision services, BPMN2, DMN, CMMN, BPM
+description: jBPM is a Open Source toolkit for building business applications backed by solid business automation capabilities.
```
Comment (MarianMacik) [inline _config/site.yml]: article should be *an* . Maybe nitpicking but looks more professional :)

**Comment by mswiderski** [comment]
updates done

File: product/services.html.haml [hunk 4]
```diff
@@ -28,11 +28,11 @@ title: Services
...
+  == Community and enterprise comparison
 
   The community download delivers everything you would expect from an Open Source project,
-  including all features. The product subscription fills the missing gaps for enterprise use.
-  Both community and product use the same code base.
+  including all features. The enterprise subscription fills the missing gaps for enterprise use.
+  Both community and enterprise use the same code base.
 
   |===
   | |Community |Product subscription
```
Comment (MarianMacik) [inline product/services.html.haml]: Enterprise subscription

File: download/download.html.haml [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+      |*Testing* | |
+      |Unit testing |image:checkYes.png[Yes] |image:checkYes.png[Yes] On certification matrix
+      |Integration testing |image:checkYes.png[Yes] |image:checkYes.png[Yes] On certification matrix
+      |Stress testing |image:checkYes.png[Yes] |image:checkYes.png[Yes]
+      |Performance testing and sizing |image:checkNo.png[No] |image:checkYes.png[Yes]
+      |QA certification |image:checkNo.png[No] |image:checkYes.png[Yes] Cross matrix of OS's, JDK's, application containers, databases, ... See https://access.redhat.com/articles/3354301[supported configurations].
+
+      |*Help* | |
+      |Roadmap impact |image:checkYes.png[Yes] Use link:../community/getHelp.html[our forum], #{site.issueTracker}[issue tracker] or send a link:../code/sourceCode.html[Pull Request] |image:checkYes.png[Yes] Prioritized above community requests
+      |Public questions |image:checkYes.png[Yes] Use link:../community/getHelp.html[StackOverflow] |image:checkYes.png[Yes] Or call 24/7 support instead
```
Comment (MarianMacik) [inline download/download.html.haml]: Do we want to prioritize StackOverflow or Google Groups?

**Pull request merged**